src/Model/Translatable/TranslationPropertiesTrait.php

```diff
@@ -15,7 +15,7 @@ trait TranslationPropertiesTrait
 
     /**
      * Will be mapped to translatable entity by TranslatableSubscriber
-     * @var TranslatableInterface
+     * @var mixed
```

Comment:
Prevent this issue:

![Capture d’écran de 2020-02-04 07-21-33](https://user-images.githubusercontent.com/2103975/73719220-ff894a80-471e-11ea-8172-1521ff910ecd.png)

Not a big fan either, but this property is only used by `getTranslatable()` which has:
- php return type to `TranslatableInterface`
- phpstan return type to `TranslatableInterface<Translatable>`

Reply:
Same thing, must be reverted to keep contract